### PR TITLE
Add missing GRR legacy names

### PR DESCRIPTION
**Pull request by destijl**
Also exempt legacy.yaml from duplicate checking.
Apply yapf python formatter.

---
This change is [Reviewable](https://reviewable.io/reviews/forensicartifacts/artifacts/185)

### CODE REVIEW

**Comment by destijl** [comment]
@joachimmetz

File: tools/validator.py [hunk 1]
```diff
@@ -24,8 +24,8 @@ def __init__(self):
     self._artifact_registry = registry.ArtifactDefinitionsRegistry()
     self._artifact_registry_key_paths = set()
 
-  def _HasDuplicateRegistryKeyPaths(
-      self, filename, artifact_definition, source):
+  def _HasDuplicateRegistryKeyPaths(self, filename, artifact_definition,
+                                    source):
```
Comment (joachimmetz) [inline tools/validator.py]: qq. why deviate from the style?

File: tools/validator.py [hunk 2]
```diff
@@ -80,14 +80,18 @@ def CheckFile(self, filename):
         for source in artifact_definition.sources:
           if source.type_indicator in (
               definitions.TYPE_INDICATOR_WINDOWS_REGISTRY_KEY):
-            if self._HasDuplicateRegistryKeyPaths(
-                filename, artifact_definition, source):
+
+            # Exempt the legacy file from duplicate checking because it has
+            # duplicates intentionally.
+            if (filename != os.path.join('definitions', 'legacy.yaml') and
```
Comment (joachimmetz) [inline tools/validator.py]: I opt not to embed function calls in other logic too much makes debugging hard
legacy_yaml_filename = os.path.join('definitions', 'legacy.yaml')
maybe even make this a class constant?

if (filename != legacy_yaml_filename and self._HasDuplicateRegistryKeyPaths(
   ...

File: tools/validator.py (hunk 1 above)
Comment (joachimmetz) [inline tools/validator.py]: per discussion: https://github.com/ForensicArtifacts/artifacts/issues/186

**Comment by joachimmetz** [comment]
LGTM

**Pull request merged**